private_set_intersection/python/psi_bindings.cpp

```diff
@@ -52,18 +98,49 @@ void bind(pybind11::module& m) {
       .def("set_reveal_intersection",
            &psi_proto::Request::set_reveal_intersection)
       .def("clear_reveal_intersection",
-           &psi_proto::Request::clear_reveal_intersection);
+           &psi_proto::Request::clear_reveal_intersection)
+      .def("load", [](psi_proto::Request& obj,
+                      const std::string& data) { return loadProto(obj, data); })
+      .def("save", [](const psi_proto::Request& obj) { return saveProto(obj); })
+      .def_static("Load", [](const std::string& data) {
+        psi_proto::Request obj;
+        loadProto(obj, data);
+        return obj;
+      });
   py::class_<psi_proto::Response>(m, "proto_response")
+      .def(py::init<>())
       .def("encrypted_elements_size",
            &psi_proto::Response::encrypted_elements_size)
-      .def("encrypted_elements",
-           py::overload_cast<int>(&psi_proto::Response::encrypted_elements,
-                                  py::const_))
+      .def(
+          "encrypted_elements",
+          [](const psi_proto::Response& obj) {
+            auto elements = obj.encrypted_elements();
+            std::vector<py::bytes> result;
+            for (auto& e : elements) result.push_back(e);
+
+            return result;
+          },
+          py::call_guard<py::gil_scoped_release>())
+      .def(
+          "encrypted_elements",
+          [](const psi_proto::Response& obj, size_t idx) {
+            return py::bytes(obj.encrypted_elements()[idx]);
+          },
+          py::call_guard<py::gil_scoped_release>())
       .def("add_encrypted_elements",
            py::overload_cast<const std::string&>(
                &psi_proto::Response::add_encrypted_elements))
       .def("clear_encrypted_elements",
-           &psi_proto::Response::clear_encrypted_elements);
+           &psi_proto::Response::clear_encrypted_elements)
+      .def("load", [](psi_proto::Response& obj,
+                      const std::string& data) { return loadProto(obj, data); })
+      .def("save",
+           [](const psi_proto::Response& obj) { return saveProto(obj); })
+      .def_static("Load", [](const std::string& data) {
```

Comment:
Why "Load" instead of "load"? can't it be overloaded?

Reply:
I wanted to make it clear it is a static method. 

LE: I tried, the compilation fails with: "ImportError: overloading a method with both static and instance methods is not supported;"

Reply:
I see. Yeah make sense